main.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"abuse-scanner/database"
+	"abuse-scanner/email"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"context"
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	// load env
+	_ = godotenv.Load()
+
+	// create a context
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// fetch env variables
```

Comment:
Should these be validated to not be blank?

Reply:
Will F/U with this one?